src/bin/vip-import-sql.js

```diff
@@ -121,6 +121,12 @@ const gates = async ( app, env, fileName ) => {
 		);
 	}
 
+	if ( ! env?.importStatus ) {
+		await track( 'import_sql_command_error', { errorType: 'empty-import-status' } );
```

Comment:
```suggestion
		await track( 'import_sql_command_error', { error_type: 'empty-import-status' } );
```

Reply:
Ahhm weird, I grabbed it from another spot where they were like that so fixed those as well. (819a49fc881b1c5094275ed8c2a08fe555074144) Good catch!